index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/2849147338.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <ul id="toplinks">
+            <li class="sociallinks">
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li class="toplink"><a href="#">English</a></li>
+            <li class="toplink"><a href="#">My Page</a></li>
+            <li class="toplink"><a href="#">Logout</a></li>
+        </ul>
+        <nav class= "main-nav">
+            
+            <div class="nav-logo">
+                <div class="logo-container">
+                    <a href="index.html">
+                        <img src="asset/fit-removebg-preview.png" alt="">
+                    </a>
+                </div>
+                <div class="hamburger">
+                    <i class="fa-solid fa-bars"></i>
+                </div>
+            </div>
+            <ul class="nav-link">
+               <div class="menu-closebtn"><i class="fa-solid fa-xmark"></i></div>
+               <li class ="list"> <a href="about.html">About</li></a>   
+               <li class ="list"><a href="#main-program">Program</li></a>
+               <li class ="list"><a href="#">join</li></a>
+               <li class ="list"><a href="#grid">sponsor</li></a>
+               <li class ="list"><a href="#">News</li></a>
+               <li class ="list"><a href="#" class="liste">CC campaign</li></a>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="main-top">
+          <div class="background">
+            <div class="message-top">
+             <div class="message">
+                <h3>welcome to our fitness program</h3>
+                <h1>international Fitness  <br> Submit conference <br> 2022</h1>
+               <p class="fit">To be physically fit means to be in a state of health and well-being. 
+                Physical fitness is defined as the body's ability to function efficiently and effectively in work and leisure activities, to be healthy, to resist disease and to react to emergency situations.</p>
+                <p class="date">Fri 26th - Sun 28th August Bank Holiday Weekend <br>
+               <span> @raffles city convention center, singapore and more</span></p>
+             </div>
+            </div>
+          </div>
+        </section>
+        <section class="main-bottom">
+            <h2 class="section-title" id="main-program">Main Program</h2>
```

Comment:
- [ ] You have not implemented requested change by the previous reviewer which was about background image of main program section. Kindly implement that.

Reply:
it is not compulsory nah. others have submitted successfully without putting it